server/src/dtos/person.dto.ts

```diff
@@ -113,6 +113,8 @@ export class AssetFaceWithoutPersonResponseDto {
   boundingBoxY1!: number;
   @ApiProperty({ type: 'integer' })
   boundingBoxY2!: number;
+  @ApiProperty({ type: 'string' })
+  sourceType?: string | null;
```

Comment:
This should be an enum

Reply:
Implemented by 2f2e288e01cabf285d48464403fef939b13e73a7